src/gt4py/cartesian/backend/dace_backend.py

```diff
@@ -546,6 +546,12 @@ def apply(cls, stencil_ir: gtir.Stencil, builder: "StencilBuilder", sdfg: dace.S
         with dace.config.temporary_config():
```

Comment:
```suggestion
        # To prevent conflict with 3rd party usage of DaCe config always make sure that any changes
        #  be under the temporary_config manager
        with dace.config.temporary_config():
```